priv/static/js/app.js

```diff
@@ -1,2 +1,2 @@
)return[];var e=this._buffer;return this._buffer=[],e.reduce((function(e,t){return e.concat(t)}),[])}}]),e}(),Yt=".progress-bar, .resource-usage-legend-entry",Bt=function(e){e.querySelectorAll(Yt).forEach((function(e){if(jt){var t=e.getAttribute("data-name")!==jt;e.setAttribute("data-muted",t)}else e.removeAttribute("data-muted")}))},Ft={PhxChartComponent:{mounted:function(){var e,t,n,i=this,r=this.el.parentElement.querySelector(".chart"),o=r.getBoundingClientRect(),a=Object.assign({},r.dataset,{tagged:r.dataset.tags&&""!==r.dataset.tags||!1,width:Math.max(o.width,At),height:Tt,now:new Date/1e3,refreshInterval:1e3});this.chart=new Nt(r,a),window.addEventListener("resize",(e=function(){var e=r.getBoundingClientRect();i.chart.resize(e)},n=!1,function(){n||(requestAnimationFrame(e),n=!0,setTimeout((function(){n=!1}),t))}))},updated:function(){var e=Array.from(this.el.children||[]).map((function(e){var t=e.dataset;return{x:t.x,y:+t.y,z:+t.z/1e6}}));e.length>0&&this.chart.pushData(e)},destroyed:function(){this.chart.clearTimers()}},PhxRequestLoggerCookie:{updated:function(){var e,t,n={key:(e=this).el.getAttribute("data-cookie-key"),value:e.el.getAttribute("data-cookie-value"),domain:e.el.getAttribute("data-cookie-domain")};t=n,document.cookie="".concat(t.key,"=; expires=").concat("Thu, 01 Jan 1970 00:00:00 GMT"),function(e){return"true"===e.el.getAttribute("data-cookie-enabled")}(this)&&function(e){var t="".concat(e.key,"=").concat(e.value,";samesite=strict;path=/");e.domain&&(t+=";domain=".concat(e.domain)),document.cookie=t}(n)}},PhxRequestLoggerQueryParameter:{mounted:function(){var e=this;this.el.querySelector(".btn-primary").addEventListener("click",(function(t){!function(e){if(navigator.clipboard){var t=e.value;navigator.clipboard.writeText(t)}else e.select(),e.setSelectionRange(0,99999),document.execCommand("copy")}(e.el.querySelector("textarea"));var n=e.el.querySelector(".copy-indicator");n.setAttribute("data-enabled","false"),n.offsetWidth,n.setAttribute("data-enabled","true")}))}},PhxRequestLoggerMessages:{updated:function(){if(this.el.querySelector(".logger-autoscroll-checkbox").checked){var e=this.el.querySelector("#logger-messages");e.scrollTop=e.scrollHeight}}},PhxColorBarHighlight:{mounted:function(){var e=this;this.el.setAttribute("data-highlight-enabled","true"),this.el.querySelectorAll(Yt).forEach((function(t){return t.addEventListener("click",(function(t){var n=t.currentTarget.getAttribute("data-name");jt=n===jt?null:n,Bt(e.el)}))}))},updated:function(){this.el.setAttribute("data-highlight-enabled","true"),Bt(this.el)}}},Ut=document.querySelector("html").getAttribute("phx-socket")||"/live",Jt=document.querySelector("meta[name='csrf-token']").getAttribute("content"),qt=new a.LiveSocket(Ut,i.Socket,{hooks:Ft,params:{_csrf_token:Jt}});window.addEventListener("phx:page-loading-start",(function(e){return o.a.start()})),window.addEventListener("phx:page-loading-stop",(function(e){return o.a.done()})),qt.connect(),window.liveSocket=qt}]);
```

Comment:
Please do not update this file, as we can't accept contributed app.js for security reasons. :)